Samples/ForeignAccelerometer/README.md

```diff
@@ -0,0 +1,5 @@
+# ForeignAccelerometer
```

Comment:
It was already there, but I forgot to make a README and add it to the table of contents when I wrote that example.